recipes/pipewire/all/conanfile.py

```diff
@@ -0,0 +1,439 @@
+        tc.project_options["jack"] = "disabled"
+        tc.project_options["legacy-rtkit"] = False
+        tc.project_options["libpulse"] = feature("with_pulseaudio")
+        tc.project_options["libusb"] = "disabled"
+        tc.project_options["man"] = "disabled"
+        tc.project_options["pipewire-alsa"] = feature("with_libalsa")
+        tc.project_options["pipewire-jack"] = "disabled"
+        tc.project_options["pipewire-v4l2"] = "disabled"
+        tc.project_options["opus"] = feature("with_opus")
+        tc.project_options["pw-cat"] = "enabled"
+        tc.project_options["pw-cat-ffmpeg"] = feature("with_ffmpeg")
+        tc.project_options["raop"] = feature("raop")
+        tc.project_options["readline"] = feature("with_readline")
+        tc.project_options["sdl2"] = "disabled"
+        tc.project_options["selinux"] = feature("with_selinux")
+        tc.project_options["session-managers"] = []
+        tc.project_options["sndfile"] = feature("with_libsndfile")
+        tc.project_options["spa-plugins"] = "enabled"
+        tc.project_options["sysconfdir"] = os.path.join(self.package_folder, "res")
+        tc.project_options["systemd"] = "disabled"
+        tc.project_options["systemd-user-service"] = "disabled"
+        tc.project_options["tests"] = "disabled"
+        tc.project_options["udev"] = feature("with_libudev", True)
+        tc.project_options["udevrulesdir"] = os.path.join(
+            self.package_folder, "res", "udev", "rules.d"
+        )
+        tc.project_options["vulkan"] = feature("with_vulkan")
+        tc.project_options["x11"] = feature("with_x11")
+        tc.project_options["x11-xfixes"] = feature("with_xfixes")
+        tc.generate()
+
+    def _patch_sources(self):
+        apply_conandata_patches(self)
+
+    def build(self):
+        self._patch_sources()
+        meson = Meson(self)
+        meson.configure()
+        meson.build()
+
+    @property
+    def _libpipewire_api_version_txt(self):
+        return os.path.join(self.package_folder, "res", "libpipewire-api-version.txt")
+
+    @property
+    def _libspa_api_version_txt(self):
+        return os.path.join(self.package_folder, "res", "libspa-api-version.txt")
+
+    def package(self):
+        copy(
+            self,
+            "COPYING",
+            self.source_folder,
+            os.path.join(self.package_folder, "licenses"),
+        )
+        meson = Meson(self)
+        meson.install()
+
+        libpipewire_pc_file = glob.glob(
+            os.path.join(self.package_folder, "lib", "pkgconfig", "libpipewire-*.pc")
+        )[0]
+        libspa_pc_file = glob.glob(
+            os.path.join(self.package_folder, "lib", "pkgconfig", "libspa-*.pc")
+        )[0]
+
+        libpipewire_api_version = os.path.splitext(
+            os.path.basename(libpipewire_pc_file)
+        )[0].split("-")[1]
+        libspa_api_version = os.path.splitext(os.path.basename(libspa_pc_file))[
+            0
+        ].split("-")[1]
```

Comment:
```suggestion
        pkconfig_dir = self.package_path.joinpath("lib", "pkgconfig")
        libpipewire_pc_file = next(pkconfig_dir.glob("libpipewire-*.pc"))
        libspa_pc_file = next(pkconfig_dir.glob("libspa-*.pc"))

        libpipewire_api_version = libpipewire_pc_file.stem.split("-")[1]
        libspa_api_version = libspa_pc_file.stem.split("-")[1]
```
Just a suggestion. The pathlib API is much nicer for such tasks.